src/test/java/apoc/export/csv/ExportXlsTest.java

```diff
@@ -154,9 +140,9 @@ private void assertExcelFileForGraph(String fileName) {
             Workbook wb = WorkbookFactory.create(inp);
 
             int numberOfSheets = wb.getNumberOfSheets();
-            assertEquals(Iterables.count(tx.getAllLabels()) + Iterables.count(tx.getAllRelationshipTypes()), numberOfSheets);
+            assertEquals(Iterables.count(tx.getAllLabelsInUse()) + Iterables.count(tx.getAllRelationshipTypesInUse()), numberOfSheets);
```

Comment:
is this an unrelated change?

Reply:
It's releated because at line 110 I create new labels `Customer` & `Company`, this makes fail the other tests now